### PR TITLE
fixed issue #3576

### PR DESCRIPTION
**Pull request by pdamodaran**

### CODE REVIEW

File: letsencrypt-auto-source/letsencrypt-auto [hunk 1]
```diff
@@ -1055,9 +1055,11 @@ cffi==1.10.0 \
     --hash=sha256:5576644b859197da7bbd8f8c7c2fb5dcc6cd505cadb42992d5f104c013f8a214 \
     --hash=sha256:b3b02911eb1f6ada203b0763ba924234629b51586f72a21faacc638269f4ced5
 ConfigArgParse==0.12.0 \
-    --hash=sha256:28cd7d67669651f2a4518367838c49539457504584a139709b2b8f6c208ef339
+    --hash=sha256:28cd7d67669651f2a4518367838c49539457504584a139709b2b8f6c208ef339 \
+    --no-binary pycparser
```
Comment (bmw) [inline letsencrypt-auto-source/letsencrypt-auto]: Whoops! Sorry I wasn't more clear on this. The syntax here isn't always `--no-binary pycparser`. It's `--no-binary <package_name>` where `<package_name>` is the name of the package not to use binary versions of. For example, this line should be `--no-binary ConfigArgParse`

**Comment by bmw** [comment]
For tracking purposes, this PR was originally opened at #6010.

**Comment by pdamodaran** [comment]
Oops - I had not meant to do that - I will fix in the next few days.

**Comment by bmw** [comment]
Closing in favor of #6023.

**Pull request closed without merging**